lib/tpl/dokuwiki/css/_footnotes.css

```diff
@@ -26,6 +33,11 @@ div.insitu-footnote {
 .dokuwiki div.footnotes div.fn div.content {
     display: inline;
 }
-.dokuwiki div.footnotes div.fn sup a.fn_bot {
+.dokuwiki div.footnotes div.fn a.fn_bot {
     font-weight: bold;
+	vertical-align: super;
+	font-size: 66.7%;
+}
+.dokuwiki div.footnotes div.fn a.fn_bot::after {
+	content: ')';
```

Comment:
Same here, happy to properly "fix" footnotes.

Reply:
As mentioned above, I kept the bracket (in CSS), but I made the footnote indexes leading the actual footnote now in normal size and style.

Reply:
As a third party who uses and loves both Dokuwiki and the Ad Hominem template, I prefer it without the bracket, too. Just a third party user's opinion, and not a dev/collaborator at all :)

Reply:
Let me also geek out a bit in explaining why the footnote symbols in the text are in bold (and that specific size):

50% more than the regular text is the approximate relation that most **bold** font faces have added to the stem size. By reducing the size by 2/3 and adding bold, we end up as close as possible to the size of actual superscript numbers in terms of stem width.

Here is a screenshot detail of the above:

![image](https://github.com/user-attachments/assets/7f055b7a-fb37-4f14-b73c-c4db49af4963)

This is not a perfect match, but probably as good as we can get with the limited possibilities of CSS (there is an even better option that requires a custom counter with actual superscript glyphs, but that would probably be a bit of overkill here).

It is not very relevant here, but I just wanted to point it out :-)

Reply:
> As a third party who uses and loves both Dokuwiki and the Ad Hominem template, I prefer it without the bracket, too. Just a third party user's opinion, and not a dev/collaborator at all :)

For the ad-hominem template, I am now considering all these fancy counter options. Maybe we'll have an option to use Roman numerals here, or Greek... and definitely without brackets :-) but that's to be seen at another place... ;-)